frontend/src/App.jsx

```diff
@@ -17,7 +17,7 @@ import AddRecipe from "./Pages/AddRecipe.jsx";
 import OneRecipe from "./Pages/OneRecipe.jsx";
 import UpdateRecipe from "./Pages/UpdateRecipe.jsx";
 import Contributors from "./Pages/Contributors.jsx"; // Import the Contributors component
-import PrivacyPolicy from "./Components/PrivacyPolicy.jsx";
+import LegalTerms from "./Components/LegalTerms.jsx";
 import NotFound from "./Pages/NotFound.jsx";
```

Comment:
@Saurabhchaudhary9799 why did you change the privacy policy component name to LegalTerms? If it is not for any valid reasons please rename it as it was before.

Reply:
@AlfiyaSiddique , I changed this name because there is an error generating by this . This name was changed for valid reasons.